src/views/ImportFlow.vue

```diff
@@ -124,7 +124,10 @@ class ImportFlow extends mixins(Lang, Routes) {
   fileName = ''
 
   get importFlowTitle(): string {
-    return this.ui.title.importFlow
+    if (this.ui.pages.importFlow.hasPageTitle === true) {
```

Comment:
Should the element render at all if this is false? Perhaps we should use `v-if`?

We should also use mapped state here at least or maybe even a getter. I'm not keen on having components which know so much about the structure of a config file which is frequently changing. If all of this logic was in the trees store I think this would be easier to maintain. Maybe something like this:

```
getters: {  
  getPagesConfig: ({ui}, key) => lodash.get(ui, key, null)
}
```

What do you think?

Reply:
@object-required Can you confirm that we definitely want this element to render with an empty string?

Reply:
Ah, sorry! Realise I was looking at `createFlowTitle` which is a different situation entirely. All good in that case!